recipe/conda_build_config.yaml

```diff
@@ -542,7 +542,7 @@ qt:
 readline:
   - 8.0
 r_base:
-  - 3.5.1
```

Comment:
Do you want to drop 3.5 now or later?

Reply:
Can we effort to have two builds side by side?
I think we should drop it.

Reply:
I don't think azure will be affected. Only appveyor will have a queue. Since azure will work for R packages, we can move all R packages to azure in the same migrator. @conda-forge/core, thoughts?

Reply:
ping @ocefpaf  @conda-forge/r

Reply:
I added this back @isuruf. But now I wonder if this will work, as we changed the pinning from x.x.x to x.x.

Reply:
We probably need to remove the 3.5.1 pinning I assume, isn't  it?

Reply:
This should work fine. If you want, we can change `3.5.1` to `3.5` as well.

Reply:
Ok, then feel free to merge if you think this is ready :)

I assume we will drop 3.5 in a few days weeks then.

Reply:
Makes sense. After it is clear that the migration is running smoothly, I guess it it makes sense to drop 3.5. An Azure only migration would be cool, as this seems to scale very well.

Reply:
@bgruening Just keep in mind that 3.5 will need to get dropped before we can start migrating on bioconda.

Reply:
Ok, after sleeping over it: I am fine with dropping R 3.5 now and doing a R 3.6 only migration.

Reply:
> I assume we will drop 3.5 in a few days weeks then.

@bgruening Why do we want to drop 3.5 so fast? I thought the plan was to support two versions of R at the same time (at least until the older version became too difficult to support, like what happened to 3.4.1).

> Just keep in mind that 3.5 will need to get dropped before we can start migrating on bioconda.

@dpryan79 Why does conda-forge need to stop building conda binaries for 3.5.1 for bioconda to be able to migrate?

Reply:
You don't have to drop it, but it's easier to coordinate between projects that way.

Reply:
We have enough build capacity to build for both R versions pretty easily